### PR TITLE
Fix RSC11

### PR DESCRIPTION
**Pull request by ricardopereira**
#305

### CODE REVIEW

**Comment by tcard** [comment]
LGTM

**Comment by tcard** [comment]
@ricardopereira Also, should we merge this?

**Comment by ricardopereira** [comment]
Yes, it's passing.

File: Spec/RestClient.swift [hunk 1]
```diff
@@ -165,13 +165,13 @@ class RestClient: QuickSpec {
             context("endpoint") {
                 it("should accept an options object with a host set") {
                     let options = ARTClientOptions(key: "fake:key")
-                    options.environment = "fake"
```
Comment (mattheworiordan) [inline Spec/RestClient.swift]: Why did you change this and not also include an `environment` test?  We need both, see http://docs.ably.io/client-lib-development-guide/features/#RSC11, it states `environment` OR `restHost` can be configured.  Please add an additional test

Comment (ricardopereira) [inline Spec/RestClient.swift]: @mattheworiordan done 187513e.
